modules/calib/src/calibinit.cpp

```diff
@@ -72,6 +72,7 @@
 #include "precomp.hpp"
 #include "circlesgrid.hpp"
 #include "opencv2/imgproc/imgproc_c.h"
+#include "opencv2/flann.hpp"
```

Comment:
BTW, there is no `_c.h` file even in 4.x: https://github.com/opencv/opencv/blob/4.9.0/modules/calib3d/src/calibinit.cpp#L74